cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableTest.java

```diff
@@ -98,8 +105,11 @@ public static void afterClass() throws Exception {
   @Override
   protected BufferingTable getTable(String name, ConflictDetection conflictLevel) throws Exception {
     // ttl=-1 means "keep data forever"
-    return
-      new HBaseTable(name, ConflictDetection.valueOf(conflictLevel.name()), testHBase.getConfiguration(), true);
+    DatasetSpecification spec = DatasetSpecification.builder(name, "foo")
+      .property(Table.PROPERTY_READLESS_INCREMENT, "true")
+      .property("conflict.level", conflictLevel.name())
```

Comment:
i know it wasn't introduced here, but should we also make this a constant in Table while we're at it?